lib/decoder.js

```diff
@@ -977,15 +977,32 @@ var JpegImage = (function jpegImage() {
 })();
 module.exports = decode;
 
-function decode(jpegData, useTArray) {
+function decode(jpegData, opts) {
+  var defaultOpts = {
+    useTArray: false,
+    colorTransform: true
+  };
+  if (opts) {
+    if (typeof opts === 'object') {
+      opts = Object.assign(defaultOpts, opts);
```

Comment:
I think this is failing travis since jpeg-js supports much older versions of node that lack `Object.assign`

I'm not sure how others feel about continuing support for <4, https://nodesource.com/node-by-numbers seems to suggest 0.10 and 0.12 have about ~2% share by # downloads, and supposedly many systems running 0.10/0.12 are not downloading fresh copies.

since we only have 2 options, how do you feel about something explicit?
```js
opts = {
  useTArray: typeof opts.useTArray === 'undefined' ? defaultOpts.useTArray : opts.useTArray,
  colorTransform: typeof opts.colorTransform === 'undefined' ? defaultOpts.colorTransform : opts.colorTransform,
}
```

Reply:
Good point! I updated to PR.

Any ideas for how to support 16bit JPEGs?